app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationListViewModel.kt

```diff
@@ -32,33 +37,27 @@ import javax.inject.Inject
 @HiltViewModel
 class ConversationListViewModel @Inject constructor(
     private val navigationManager: NavigationManager,
-    private val getConversations: GetConversationsUseCase,
+    private val observeConversationDetailsList: ObserveConversationListDetailsUseCase,
+    private val getPublicAsset: GetPublicAssetUseCase
 ) : ViewModel() {
 
     var state by mutableStateOf(ConversationListState())
         private set
 
     init {
         viewModelScope.launch {
-            getConversations().let {
-                when (it) {
-                    is GetConversationsUseCase.Result.Failure -> TODO("unhandled error case")
-                    is GetConversationsUseCase.Result.Success -> {
-                        it.convFlow.collect { conversations ->
-                            state = ConversationListState(
-                                newActivities = listOf(),
-                                conversations = conversationMockData(conversations.toGeneralConversationList()),
-                                missedCalls = mockMissedCalls,
-                                callHistory = mockCallHistory,
-                                unreadMentions = mockUnreadMentionList,
-                                allMentions = mockAllMentionList,
-                                unreadMentionsCount = 12,
-                                missedCallsCount = 100,
-                                newActivityCount = 1
-                            )
-                        }
-                    }
-                }
+            observeConversationDetailsList().collect { detailedList ->
+                state = ConversationListState(
+                    newActivities = listOf(),
+                    conversations = conversationMockData(detailedList.toGeneralConversationList()),
+                    missedCalls = mockMissedCalls,
+                    callHistory = mockCallHistory,
+                    unreadMentions = mockUnreadMentionList,
+                    allMentions = mockAllMentionList,
+                    unreadMentionsCount = 12,
+                    missedCallsCount = 100,
+                    newActivityCount = 1
```

Comment:
Can we extract these values into constant values ?

Reply:
This is just mock data.
Needs to be removed.